operators/argo-controller/src/charm.py

```diff
@@ -87,10 +87,11 @@ def main(self, event):
             },
         }
 
+        crd_root = "src/crds"
         crd_paths = [
-            Path(f"files/{crd_file}")
-            for crd_file in listdir("files")
-            if Path(f"files/{crd_file}").is_file()
+            Path(f"{crd_root}/{crd_file}")
+            for crd_file in listdir(crd_root)
+            if Path(f"{crd_root}/{crd_file}").is_file()
```

Comment:
Would it make sense to write this as `[yaml.safe_load(Path(f).read_text()) for f in glob('files/*.yaml')]`?